scripts/publish-gh-release-notes.py

```diff
@@ -61,7 +61,7 @@ def parse_changelog(tag_name):
 
 
 def convert_rst_to_md(text):
-    return pypandoc.convert_text(text, "md", format="rst")
+    return pypandoc.convert_text(text, "md", format="rst", extra_args=["--wrap=none"])
```

Comment:
Why not `preserve`?

Reply:
That looks good too:

```
AFTER
pytest 5.3.5 (2020-01-29)
=========================

Bug Fixes
---------

-   [\#6517](https://github.com/pytest-dev/pytest/issues/6517): Fix regression in pytest 5.3.4 causing an INTERNALERROR due to a wrong assertion.
```

Will update.

Reply:
Ok.  I think this will allow us to "force" newlines when necessary etc.